stingray/gti.py

```diff
@@ -620,6 +620,28 @@ def gti_len(gti):
     return np.sum(gti[:, 1] - gti[:, 0])
 
 
+@jit(nopython=True)
+def _check_separate(gti0, gti1):
```

Comment:
Is there a non-numba compiled version of this? Or do we require numba as a required dependency?

Reply:
If numba is not installed, we have a mocked `jit` decorator that just returns the function. It will be slow, but it will work!